tests/forte/data/readers/sst2_reader_test.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for SST2Reader.
+"""
+import json
+import os
+import unittest
+from typing import Iterator, Iterable
+
+from forte.data.data_pack import DataPack
+from forte.data.readers import SST2Reader
+from forte.pipeline import Pipeline
+from ft.onto.base_ontology import ConstituentNode
+
+class SST2ReaderTest(unittest.TestCase):
+    def setUp(self):
+        self.dataset_path = os.path.abspath(os.path.join(
```

Comment:
It seems that we don't have the sample data here, this test would probably fail, you can put one sentence sample somewhere maybe?